browser/components/atoms/Date/index.js

```diff
@@ -4,6 +4,7 @@ import moment from 'moment'
 export default class Date extends Component {
   static propTypes = {
     date: PropTypes.oneOfType([
+      PropTypes.instanceOf(moment),
```

Comment:
change the commit message to "Added `date: instanceof(moment)` to Date PropTypes"